src/web/app/components/question-types/question-edit-details-form/contribution-question-edit-details-form.component.scss

```diff
@@ -0,0 +1,3 @@
+.disabled {
+  color: gray;
+}
```

Comment:
Do not override the default bootstrap utility css classes for no reason. This makes it annoying for others who want to use the original class in this file. 

Use the pre-defined text-muted, or if that is not suitable then rename this to text-gray.